### DISCUSSION TITLE
Differences between Mini and Full release packages?

### DISCUSSION THREAD

**Discussion by maphew** [cmderdev/cmder#2865] [Q&A] [answered]
Is there a page that summarizes the differences between the Mini and Full releases?

I found this paragraph in https://github.com/cmderdev/cmder/blob/master/README.md#installation:

> You may use your favorite version of Git by including its path in the %PATH% environment variable. Moreover, the Mini edition of Cmder (found on the [downloads page](https://github.com/cmderdev/cmder/releases)) excludes any vendored Git binaries.

Are there other differences?

**Comment by DRSDavidSoft** [comment, marked as answer]
Hi there, this is also explained over at #2715, but essentially the packages are the same, the mini one just excludes Git for Windows. In terms of shells, profile configuration and other areas the mini version includes the same stuff as the full one.

**Reply by maphew** [reply]
Thanks! 

fyi, I searched the Wiki and Discussions before posting but did not search Issues.
